wrapper/src/test/java/integration/refactored/host/TestEnvironment.java

```diff
@@ -617,12 +649,62 @@ private static String getEnvironmentInfoAsString(TestEnvironment env) {
 
   public void runTests(String taskName) throws IOException, InterruptedException {
     final ContainerHelper containerHelper = new ContainerHelper();
-    containerHelper.runTest(this.testContainer, taskName);
+
+    if (this.info.getRequest().getFeatures().contains(TestEnvironmentFeatures.RUN_HIBERNATE_TESTS_ONLY)) {
+      final Long exitCode = containerHelper.runCmdInDirectory(
+          this.testContainer,
+          "/app/hibernate-orm",
+          buildHibernateCommands(false));
+      containerHelper.runCmd(this.testContainer, "./collect_test_results.sh");
+      assertEquals(0, exitCode, "failed to run Hibernate ORM tests");
+    } else {
+      containerHelper.runTest(this.testContainer, taskName);
+    }
   }
 
   public void debugTests(String taskName) throws IOException, InterruptedException {
     final ContainerHelper containerHelper = new ContainerHelper();
     containerHelper.debugTest(this.testContainer, taskName);
+
+    if (this.info.getRequest().getFeatures().contains(TestEnvironmentFeatures.RUN_HIBERNATE_TESTS_ONLY)) {
+      final Long exitCode = containerHelper.runCmdInDirectory(
```

Comment:
does this allow us to actually debug the hibernate tests? Looks like it just runs them

Reply:
Yes. The method runs whatever command we supply and the buildHibernateCommands(true) determines whether we are supplying a debug command or a run command.